src/bentoml/_internal/frameworks/transformers.py

```diff
@@ -812,7 +843,14 @@ def import_model(
 
     if model is None:
         with init_empty_weights():
-            model = pretrained_model_class(config=config)
+            if is_auto_class:
+                model = pretrained_model_class.from_config(
```

Comment:
probably add a note, and link to marc response on accelerate https://github.com/huggingface/accelerate/issues/2163#event-11018561032 explaining why we do this.